### ISSUE TITLE
Cannot declare class OCA\Secrets\Cron\CleanupExpiredSecrets, because the name is already in use

### ISSUE THREAD

**Issue by zfkdnop** [theCalcaholic/nextcloud-secrets#7]
**From the Nextcloud Logging interface:**
`Cannot declare class OCA\Secrets\Cron\CleanupExpiredSecrets, because the name is already in use at /home/USER/DOMAIN/apps/secrets/lib/Cron/SecretCleanup.php#0`

**From PHP logs:**
```
Dec  9 05:35:18 HOSTNAME systemd[1]: Started Nextcloud cron.php job.
Dec  9 05:35:18 HOSTNAME php[331241]: PHP Warning:  Missing arginfo for uploadprogress_get_info() in Unknown on line 0
Dec  9 05:35:18 HOSTNAME php[331241]: PHP Warning:  Missing arginfo for uploadprogress_get_contents() in Unknown on line 0
Dec  9 05:35:18 HOSTNAME php[331241]: PHP Fatal error:  Cannot declare class OCA\Secrets\Cron\CleanupExpiredSecrets, because the name is already in use in /home/USER/DOMAIN/apps/secrets/lib/Cron/SecretCleanup.php on line 0
Dec  9 05:35:18 HOSTNAME php[331241]: PHP Stack trace:
Dec  9 05:35:18 HOSTNAME php[331241]: PHP   1. {main}() /home/USER/DOMAIN/cron.php:0
Dec  9 05:35:18 HOSTNAME php[331241]: PHP   2. OC\BackgroundJob\JobList->getNext($onlyTimeSensitive = FALSE) /home/USER/DOMAIN/cron.php:144
Dec  9 05:35:18 HOSTNAME php[331241]: PHP   3. OC\BackgroundJob\JobList->buildJob($row = ['id' => '8136', 'class' => 'OCA\\Secrets\\Cron\\SecretCleanup', 'argument' => 'null', 'last_run' => '0', 'last_checked' => '1670542487', 'reserved_at' => '1670542487', 'execution_duration' => '0', 'argument_hash' => '37a6259cc0c1dae299a7866489dff0bd', 'time_sensitive' => '1']) /home/USER/DOMAIN/lib/private/BackgroundJob/JobList.php:241
Dec  9 05:35:18 HOSTNAME php[331241]: PHP   4. OC\ServerContainer->query($name = 'OCA\\Secrets\\Cron\\SecretCleanup', $autoload = *uninitialized*) /home/USER/DOMAIN/lib/private/BackgroundJob/JobList.php:303
Dec  9 05:35:18 HOSTNAME php[331241]: PHP   5. OC\AppFramework\Utility\SimpleContainer->query($name = 'OCA\\Secrets\\Cron\\SecretCleanup', $autoload = TRUE) /home/USER/DOMAIN/lib/private/ServerContainer.php:161
Dec  9 05:35:18 HOSTNAME php[331241]: PHP   6. OC\AppFramework\Utility\SimpleContainer->resolve($name = 'OCA\\Secrets\\Cron\\SecretCleanup') /home/USER/DOMAIN/lib/private/AppFramework/Utility/SimpleContainer.php:133
Dec  9 05:35:18 HOSTNAME php[331241]: PHP   7. ReflectionClass->__construct($objectOrClass = 'OCA\\Secrets\\Cron\\SecretCleanup') /home/USER/DOMAIN/lib/private/AppFramework/Utility/SimpleContainer.php:114
Dec  9 05:35:18 HOSTNAME php[331241]: PHP   8. Composer\Autoload\ClassLoader->loadClass($class = 'OCA\\Secrets\\Cron\\SecretCleanup') /home/USER/DOMAIN/lib/private/AppFramework/Utility/SimpleContainer.php:114
Dec  9 05:35:18 HOSTNAME php[331241]: PHP   9. Composer\Autoload\includeFile($file = '/home/USER/DOMAIN/apps/secrets/lib//Cron/SecretCleanup.php') /home/USER/DOMAIN/lib/composer/composer/ClassLoader.php:428
Dec  9 05:35:18 HOSTNAME systemd[1]: nextcloudcron.service: Main process exited, code=exited, status=255/n/a
Dec  9 05:35:18 HOSTNAME systemd[1]: nextcloudcron.service: Failed with result 'exit-code'.
```

**cat /etc/systemd/system/nextcloudcron.service**

> [Unit]
> Description=Nextcloud cron.php job
> 
> [Service]
> User=USER
> ExecStart=php -f /home/USER/DOMAIN/cron.php
> KillMode=process

cron.php is Nextcloud's cron.php...

**uname -a;php -v;cat /etc/os-release** 

> Linux HOSTNAME.DOMAIN 4.18.0-425.3.1.el8.x86_64 #1 SMP Tue Nov 8 14:08:25 EST 2022 x86_64 x86_64 x86_64 GNU/Linux
> PHP Warning:  Missing arginfo for uploadprogress_get_info() in Unknown on line 0
> PHP Warning:  Missing arginfo for uploadprogress_get_contents() in Unknown on line 0
> PHP 8.0.26 (cli) (built: Dec  1 2022 03:39:41) ( NTS )
> Copyright (c) The PHP Group
> Zend Engine v4.0.26, Copyright (c) Zend Technologies
>     with Zend OPcache v8.0.26, Copyright (c), by Zend Technologies
>     with Xdebug v3.1.4, Copyright (c) 2002-2022, by Derick Rethans
> NAME="AlmaLinux"
> VERSION="8.7 (Stone Smilodon)"
> ID="almalinux"
> ID_LIKE="rhel centos fedora"
> VERSION_ID="8.7"
> PLATFORM_ID="platform:el8"
> PRETTY_NAME="AlmaLinux 8.7 (Stone Smilodon)"
> ANSI_COLOR="0;34"
> LOGO="fedora-logo-icon"
> CPE_NAME="cpe:/o:almalinux:almalinux:8::baseos"
> HOME_URL="https://almalinux.org/"
> DOCUMENTATION_URL="https://wiki.almalinux.org/"
> BUG_REPORT_URL="https://bugs.almalinux.org/"
> 
> ALMALINUX_MANTISBT_PROJECT="AlmaLinux-8"
> ALMALINUX_MANTISBT_PROJECT_VERSION="8.7"
> REDHAT_SUPPORT_PRODUCT="AlmaLinux"
> REDHAT_SUPPORT_PRODUCT_VERSION="8.7"
>

**Comment by theCalcaholic** [comment]
Hi, thank you for the report. A fix is already implemented, i just didn't find the time to release it yet

**Comment by theCalcaholic** [comment]
The cron job should now be fixed in version 1.2.0 (thanks @vitormattos)
